includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -641,14 +712,14 @@ public function stripe_connect_field( $value, $option_value ) {
 					if ( give_stripe_is_connected() ) :
 						$site_url            = get_site_url();
 						$stripe_user_id      = give_get_option( 'give_stripe_user_id' );
-						$modal_title         = __( '<strong>You are connected! Now this is important: Please now configure your Stripe webhook to finalize the setup.</strong>', 'give' );
+						$modal_title         = '<strong>' . esc_html__( 'You are connected! Now this is important: Please now configure your Stripe webhook to finalize the setup.', 'give' ) . '</strong>';
 						$modal_first_detail  = sprintf(
 							'%1$s %2$s',
 							__( 'In order for Stripe to function properly, you must add a new Stripe webhook endpoint. To do this please visit the <a href=\'https://dashboard.stripe.com/webhooks\' target=\'_blank\'>Webhooks Section of your Stripe Dashboard</a> and click the <strong>Add endpoint</strong> button and paste the following URL:', 'give' ),
 							"<strong>{$site_url}?give-listener=stripe</strong>"
 						);
 						$modal_second_detail = __( 'Stripe webhooks are required so GiveWP can communicate properly with the payment gateway to confirm payment completion, renewals, and more.', 'give' );
-						$can_display = ! empty( $_GET['stripe_access_token'] ) ? '0' : '1';
+						$can_display         = ! empty( $_GET['stripe_access_token'] ) ? '0' : '1';
```

Comment:
we will get boolean from `empty` function which we can use later in logic instead of `0` or `1`